setup.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+from __future__ import division, absolute_import, print_function
+
+import os
+import sys
+from setuptools import find_packages, setup
+from numpy.distutils.core import setup
+from numpy.distutils.misc_util import Configuration
```

Comment:
The `setup.py` handle the build and compilation of the extensions thanks to `numpy.distutils.misc_util.Configuration`.